sdk/documentintelligence/azure-ai-documentintelligence/tsp-location.yaml

```diff
@@ -0,0 +1,5 @@
+directory: specification/cognitiveservices/DocumentIntelligence
+commit: 763e567abdb2154599a6d397444e60affc9cee17
+additionalDirectories: []
+repo: Azure/azure-rest-api-specs-pr
```

Comment:
You should be able to update these values to the ones in the public repo if the tsp PR was already merged in

Reply:
Yup, the tsp PR is currently in open, will update once it merged.